### PR TITLE
LPS-156784 Fix Poshi Automation - master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-156784

@david-gutierrez-mesa 
@Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Icon.path [hunk 1]
```diff
@@ -57,7 +57,7 @@
 </tr>
 <tr>
 	<td>ASTERISK_DATE_FIELD</td>
-	<td>//label[contains(.,'${key_fieldFieldLabel}')]//span[*[name()='svg'][contains(@class,'icon-asterisk')]]</td>
+	<td>//label[contains(.,'${key_fieldFieldLabel}')]//span[*[name()='svg'][contains(@class,'icon-asterisk')]] | //label[contains(.,'${key_fieldFieldLabel}')][*[name()='svg'][contains(@class,'icon-asterisk')]]</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Icon.path:60]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/TextInput.testcase [hunk 2]
```diff
@@ -550,8 +550,8 @@ definition {
 				siteName = "Test Site Name");
 
 			AssertNotVisible(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/TextInput.testcase:552]: AssertElementNotPresent

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/TextInput.testcase [hunk 3]
```diff
@@ -660,8 +660,8 @@ definition {
 			ControlMenu.viewHeaderTitle(headerTitle = "Display Page Name");
 
 			AssertNotVisible(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/TextInput.testcase:662]: AssertElementNotPresent

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comment? Thanks

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**